dpxdt/tools/site_diff.py

```diff
@@ -23,6 +23,8 @@
     --release_client_id=<your api key> \
     --release_client_secret=<your api secret> \
     --crawl_depth=1 \
+    --width=320
```

Comment:
For this command to work you need a `\` character at the end of this line and the next one.